net/suricata-update/files/install_suricata_update

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh /etc/rc.common
+/usr/bin/python3 -m pip install --upgrade pip
```

Comment:
uhhhh what?

Reply:
It ensures pip3 is up to date, since it's currently outdated in master.  Not a great idea?